platform/darwin/src/MGLTileSet.h

```diff
@@ -14,6 +14,16 @@ typedef NS_ENUM(NSUInteger, MGLTileSetScheme) {
  configuration for those strings. It can be passed to an `MGLVectorSource` or
  `MGLRasterSource` instead of an `NSURL` representing a TileJSON URL to create a 
  source.
+ 
+ Tile URL template strings are a loose open standard for referring to custom 
```

Comment:
ac1bfafe2894e15c6b24c75b59060a11c8733884 tidies this up a bit and includes better examples.